nilearn/plotting/__init__.py

```diff
@@ -1,61 +1,7 @@
 """Plotting code for nilearn."""
 
 # Original Authors: Chris Filo Gorgolewski, Gael Varoquaux
-import importlib
-import warnings
-
-OPTIONAL_MATPLOTLIB_MIN_VERSION = "3.3.0"
-
-###############################################################################
-# Make sure that we don't get DISPLAY problems when running without X on
-# unices
-
-
-def _set_mpl_backend():
-    # We are doing local imports here to avoid polluting our namespace
-    try:
-        import matplotlib
-    except ImportError:
-        if importlib.util.find_spec("pytest") is not None:
-            from .._utils.testing import skip_if_running_tests
-
-            # No need to fail when running tests
-            skip_if_running_tests("matplotlib not installed")
```

Comment:
I've removed these lines while moving the function to `nilearn._utils.helpers.py` as the tests of plotting package is already ignored by `conftest.py`